pygfx/renderers/wgpu/_renderer.py

```diff
@@ -195,6 +196,9 @@ def __init__(
         self.blend_mode = blend_mode
         self.sort_objects = sort_objects
 
+        # Lights, Fogs，etc.
+        self._render_state = {}
```

Comment:
I think you can simplify your code a little bit by initializing like this:

```suggestion
        self._render_state = {
            "lights": [],
        }
```